src/utils.coffee

```diff
@@ -44,13 +44,18 @@ class Utils
     result = [result] unless result instanceof Array
     _.compact(result)
 
-  @wrapObjects: (array) ->
+  @wrapObjects: (array, options = {}) ->
     output = []
     _(array).each (elem) =>
       if elem.constructor == Object
         for key, value of elem
           o = {}
-          o[key] = @wrapObjects(if value instanceof Array then value else [value])
+
+          if options.ignoreCollections && elem[key] instanceof Backbone.Collection
```

Comment:
Shouldn't the guard then be a few lines above? Explicitly avoid iterating through each key-value pair if it's a backbone collection? i.e. `if elem.constructor == Object` needs to do the right thing.